tox.ini

```diff
@@ -16,7 +16,7 @@ setenv =
 
 [testenv:lint]
 deps = flake8
-commands = flake8 --max-line-length 120 --max-complexity 10 scripts src tests
+commands = flake8 --max-line-length 120 --max-complexity 15 scripts src tests
```

Comment:
I did not know that the linter uses function complexity measures and I do not like it

Reply:
I think i should replace flake8 entirely with the more friendly linter we use elsewhere.

Reply:
Looks like we do use it elsewhere with some line length and indentation rules disabled https://github.com/wellcomecollection/storage-service/blob/master/makefiles/formatting.Makefile#L7 - perhaps it's an old version without the complexity stuff?

Reply:
I’m gonna guess it’s gone the other way, and this is the older version.

All the build tooling is lifted from Hypothesis and hyper-h2, and the latter cares about this, IMO correctly for that project: https://github.com/python-hyper/hyper-h2/blob/master/tox.ini#L14-L17

The other linting-adjacent tool we use is black, which is more an autoformatter than a linter. Unless @kenoir is thinking of something else?